js/script.js

```diff
@@ -1,14 +1,21 @@
-(function () {
-    "use strict"
-    
-    document.getElementById("btn-hamburguer").onclick = function(){
+(function() {
+    (function () {
+        if (window.outerWidth < 600) {
+            let tamaño = window.outerWidth + "px " + "100%;";
+            let cadena = "background-size: ";
+            cadena += tamaño;

```

Comment:
Codacy has a fix for the issue: [Replace `············cadena·+=·tamaño;␍` with `······cadena·+=·tamaño;`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456239)
```suggestion
      cadena += tamaño;
```